Projects.js

```diff
@@ -176,20 +176,21 @@ function button(id) {
                </p>
           </div>
             <div class="card_tags">
-             <div class="modeltags">
-          <div class="tag">
+              <div class="modeltags">
+               <div class="tag">
                  <span class="small">${card.tags.html}</span>
               </div>
               <div class="tag">
                <span class="small">${card.tags.css}</span>
-                 </div>
-                <div class="tag">
-              <span class="small"
+              </div>
+              <div class="tag">
+                <span class="small"
                  >${card.tags.javascript}</span
                        >
-               </div>
-                 </div>
-             <div class="card_action">
+              </div>
+            </div>
+            <hr class="hr">
+            <div class="card_action">
```

Comment:
As suggested by the previous reviewer, kindly add a horizontal line and style it as shown in the figma design given , it's always good that your design looks close to the template given to show good attention to detail 😄 

Figma design :
![image](https://github.com/sheezu900/My_Potfolio/assets/20537712/5e2a3afc-d68b-4c04-a42e-5a418210864f)

Your design :
![image](https://github.com/sheezu900/My_Potfolio/assets/20537712/6369858c-b647-4e10-b015-50481d0658e9)